corehq/util/datadog/utils.py

```diff
@@ -13,6 +13,26 @@
 datadog_metric_logger = logging.getLogger('datadog-metrics')
 
 WILDCARD = '*'
+DATADOG_USER_MAX_FORMS_LIMIT = 300
+DATADOG_MOBILE_USER_COUNT = 'commcare.mobile_user_count'
+DATADOG_EXCEEDING_MAX_FORMS_COUNT = 'commcare.users_gt_{}_forms'.format(DATADOG_USER_MAX_FORMS_LIMIT)
```

Comment:
rename to `commcare.hubspot.domains_with_forms_gt_threshold`.

If the threshold changes we don't want to change the metric name.